### DISCUSSION TITLE
Browser with Tabcontrol

### DISCUSSION THREAD

**Discussion by Fenish** [cefsharp/CefSharp#3472] [General]
when i try to use tabcontrol with chromium browser, Im getting this error
https://i.imgur.com/FyuoU6D.png

**Comment by amaitland** [comment]
Please answer questions listed at https://github.com/cefsharp/CefSharp/wiki/Questions-&-Support#when-asking-a-question-please-include-the-following

**Comment by Fenish** [comment]
**What version are you using?**
88.2.90.0
**Are you using WPF/WinForms/OffScreen?**
winforms
**Are you targeting .Net Core? If yes which version?**
4.5.2
**Os**
Windows 7

**Reply by amaitland** [reply]
I still need more information, is it a runtime or design time exception you are seeing?

Are you targeting `x86/x64/AnyCPU`?
